### ISSUE TITLE
About get centroids from training,Calculate the loss function related to the distance from the center of mass to the horizontal and vertical

### ISSUE THREAD

**Issue by Jinghan-Wang** [vqdang/hover_net#49]
Hi，Thanks for your excellent papaer and code！
          Learned from the paper  'HoVer-Net: Simultaneous Segmentation and Classification of Nuclei in Multi-Tissue Histology Images', that leverages the instance-rich information encoded within the vertical and horizontal distances of nuclear pixels to their centres of mass。I know how to predicted hv-map (pred_hv) ,but I don‘t know how to predicted centroid in training, because the loss funtion  is related to the distance from the center of mass to the horizontal and vertical。

![Uploading image.png…]()
How to get centroid? As shown in the black part . Where should I find the relevant predicted centroid distance codes.
Looking forward to your reply, thank you.

**Comment by JinghanWang041315** [comment]
Supplementary Figure（HoVer-Net  fig3）
![image](https://user-images.githubusercontent.com/53046658/83989891-6b541c00-a97a-11ea-8de1-d858be74fba5.png)

**Comment by Jinghan-Wang** [comment]
I have solved this problem，Your previous guidance to others，，https://github.com/vqdang/hover_net/issues/7#issuecomment-506961499.

**Comment by vqdang** [comment]
Close because user self-resolved in email, for others reference, please check #7 
Feel free to open another issue if you have any questions.
